R/parsed.R

```diff
@@ -1,3 +1,15 @@
+#' Parse and pre-process character vector
+#'
+#' @description The function obtains detailed parse information for `text` via
+#'   [utils::getParseData] and does some minimal pre-processing by calling
```

Comment:
Maybe it even make sense to extend the style guide with regard to `roxygen2` documentation. I opened an issue in the [tidyverse style](https://github.com/tidyverse/style/issues/21) repo.